.env.example

```diff
@@ -1,5 +1,16 @@
+TWILIO_ACCOUNT_SID=ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+TWILIO_PHONE_NUMBER=+1XXXYYYZZZZ
 
-export TWILIO_ACCOUNT_SID=Your-Account-SID
-export TWILIO_AUTH_TOKEN=Your-Twilio-Authy-Token
-export TWILIO_APP_SID=Your-Twilio-APP-SID
-export TWILIO_PHONE_NUMBER=Your-Twilio-Phone-Number
+# SID of your TwiML Application
+# https://www.twilio.com/console/voice/twiml/apps
+# OR
+# twilio api:core:applications:create --friendly-name=voice-client-javascript
+TWILIO_APP_SID=APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+# Your REST API Key information
+# https://www.twilio.com/console/project/api-keys
+# OR
+# twilio api:core:keys:create --friendly-name=voice-client-javascript -o json
+# NOTE: Make sure to copy the secret, it'll will only be displayed once
+API_KEY=SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Can we turn these into `TWILIO_API_KEY` and `TWILIO_API_SECRET` for consistency?